src/endpoint/s3/s3_bucket_policy_utils.js

```diff
@@ -110,12 +110,13 @@ const predicate_map = {
 
 const condition_fit_functions = {
```

Comment:
please add test to test_s3_bucket_policy.js

Reply:
done.